tests/framework/info.go

```diff
@@ -53,3 +56,28 @@ func GetEvents(rm ResourceManager, namespace string) string {
 	}
 	return returnEvents
 }
+
+// GetBuildInfo returns the build information.
+func GetBuildInfo() (commitHash string, commitTime string, dirtyBuild string) {
+	commitHash = "unknown"
+	commitTime = "unknown"
+	dirtyBuild = "unknown"
+
+	info, ok := debug.ReadBuildInfo()
+	if !ok {
```

Comment:
Can we do

```
if ok {
for _, kv := range info.Settings {
		switch kv.Key {
		case "vcs.revision":
			commitHash = kv.Value
		case "vcs.time":
			commitTime = kv.Value
		case "vcs.modified":
			dirtyBuild = kv.Value
		}
	}
}

instead?

Reply:
for context, I copied it from https://github.com/nginxinc/nginx-gateway-fabric/blob/main/cmd/gateway/commands.go#L508-L529 so we don't have to import the package, but for me
```
	if !ok {
		return
	}
```
makes it clearer, similar to 
```
	if err != nil {
		return
	}
```

Reply:
i see ok thanks we can keep it as it is!